Frontend/src/common/interfaces/Pagination.tsx

```diff
@@ -0,0 +1,65 @@
+import React from "react";
```

Comment:
You're storing this component in the `interfaces` directory, it should probably go into `components/common`.